config/odp-linux-generic.conf

```diff
@@ -115,6 +115,10 @@ pktio: {
 	# (zero-copy DPDK, loop and ipc) that do not copy data ignore this
 	# option.
 	pktin_frame_offset = 0
+
+	# Pool size allocated for potential completion events for transmitted and
+	# dropped packets. Separate pool for different packet IO instances.
+	num_tx_compl_evs = 1024
```

Comment:
This could be "tx_compl_pool_size", which would high light that it's not total number of events, but number of events per pool/interface.

Reply:
Renamed in v5.